reporter/src/main/kotlin/reporters/opossum/OpossumReporter.kt

```diff
@@ -380,7 +380,9 @@ class OpossumReporter : Reporter {
                 val rootForScope = resolvePath(definitionFilePath, scopeName)
                 val dependencies = ortResult.dependencyNavigator.directDependencies(project, scopeName).toList()
 
-                if (dependencies.isNotEmpty()) addAttributionBreakpoint(rootForScope)
+                if (dependencies.isNotEmpty()) {
+                    addAttributionBreakpoint(rootForScope)
+                }
```

Comment:
Why? According to our (unwritten 😁) rules `if`-one-liners are fine, even if there was an `else`, as [Kotlin has no ternary operator](https://kotlinlang.org/docs/control-flow.html#if-expression).

Reply:
According to the unwritten rules I would guess both is fine.
I did this because I found it to be a tad more readable as the right hand side
is not super short. Anyhow, if you prefer the previous version I can drop this commit.

Reply:
Then I'd indeed prefer to drop it as IMO there's no strong reason for the change.